beacon_node/network/src/attestation_service/tests/mod.rs

```diff
@@ -20,7 +20,7 @@ mod tests {
     use tempfile::tempdir;
     use types::{CommitteeIndex, EthSpec, MinimalEthSpec};
 
-    const SLOT_DURATION_MILLIS: u64 = 200;
+    const SLOT_DURATION_MILLIS: u64 = 400;
```

Comment:
@AgeManning had to increase this to make the `test_same_subnet_unsubscription` pass. 

In the test, we are trying to create 2 validator subscriptions with the same `subnet_id` but different slots. 
The issue is that `slots_per_epoch` for `MinimalEthSpec` is just 8. So `compute_subnet_id` gives different results when `current_slot`(which is random) is >= 7. By increasing the `SLOT_DURATION`, i'm hoping `current_slot` will never go greater than that. Keen to see if there's a better way.

Reply:
Yeah some of these tests probably need to be re-worked. 
Happy for a hack fix at the moment